mailet/base/src/test/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -188,10 +199,6 @@ public void setAttribute(String name, Serializable object) {
         attributes.put(name,object);
     }
 
-    public void storeMail(MailAddress sender, MailAddress recipient, MimeMessage msg) throws MessagingException {
```

Comment:
Not used.... Moreover it does not belong to the MailContext interface. And it generated a warning in intelliJ...